docs/releases/5.2.txt

```diff
@@ -178,7 +178,12 @@ Migrations
 Models
 ~~~~~~
 
-* ...
+* :attr:`~django.db.models.UniqueConstraint.violation_error_code` and 
```

Comment:
This is not really a new feature, so might be better to move this to the `Backwards incompatible changes in 5.2` section